### PR TITLE
PL updates for Actions and changing Icon Square to Fill Icons

### PR DESCRIPTION
**Pull request by line47**
General improvements to the PL.
Updating naming and for icon square to fill icons.
Separating the actions into sections (button, button outline, link)

### CODE REVIEW

File: docs/src/components/components/action/button/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+In addition, add any of the following classes to the button to style the button
+based on status.
+
+| Class   | Description |
+| ------   | -----------|
+| `action-primary usa-button-primary` | When to use |
+| `action-base usa-button-base`       | When to use |
+| `action-warning usa-button-warning`  | When to use |
+| `action-cautious usa-button-cautious`    | When to use |
+| `action-finish usa-button-finish`    | When to use |
```
Comment (thisisdano) [inline docs/src/components/components/action/button/README.md]: `warning`, `primary`, `finish`, and `base` are the allowed values in the Action `style` prop. I think `cautious` can be nixed.

File: docs/src/components/components/action/button-outline/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+In addition, add any of the following classes to the button to style the button
+based on status.
+
+| Class   | Description |
+| ------   | -----------|
+| `action-primary` | When to use |
+| `action-base`       | When to use |
+| `action-warning`  | When to use |
+| `action-cautious`    | When to use |
+| `action-finish`    | When to use |
```
Comment (thisisdano) [inline docs/src/components/components/action/button-outline/README.md]: `warning`, `primary`, `finish`, and `base` are the allowed values in the Action `style` prop. I think `cautious` can be nixed.

File: docs/src/components/components/action/link/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+To use the **link** component,  add the classes `action action-link` to the `a`
+element.
+
+| Class   | Description |
+| ------   | -----------|
+| `action-primary` | When to use |
+| `action-base`       | When to use |
+| `action-warning`  | When to use |
+| `action-cautious`    | When to use |
```
Comment (thisisdano) [inline docs/src/components/components/action/link/README.md]: `warning`, `primary`, `finish`, and `base` are the allowed values in the Action `style` prop. I think `cautious` can be nixed.

File: docs/src/components/components/fill-icons/README.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+- Follow the rules defined in the normal [icons](/components/detail/icon--default)
+  instructions for adding a new icon.
+
+## Caveats
+
+In order to make the fill icons correctly color based on their color class, the
+icon has to be made up of only fills and likely cannot include any white sections
+that overlap with other sections. Due to this limitation, there are currently
+some fill icons in the cloud.gov system that are not yet optimized to be colored,
+such as the `org` icon.
```
Comment (thisisdano) [inline docs/src/components/components/fill-icons/README.md]: I rebuilt the org icon a while back and it's now OK

File: docs/src/components/components/fill-icons/fill-icons.config.yml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-title: Icon Square
+title: Filll Icons
```
Comment (thisisdano) [inline docs/src/components/components/fill-icons/fill-icons.config.yml]: typo

File: docs/src/components/components/icon-square/README.md [hunk 6]
```diff
@@ -69,4 +69,3 @@ icon has to be made up of only fills and likely cannot include any white section
 that overlap with other sections. Due to this limitation, there are currently
 some fill icons in the cloud.gov system that are not yet optimized to be colored,
 such as the `org` icon.
```
Comment (thisisdano) [inline docs/src/components/components/icon-square/README.md]: See previous. Org icon is now ok.

**Review by thisisdano — CHANGES_REQUESTED** [review]
Couple quick changes, but otherwise OK to me

File: docs/src/components/components/action/button/README.md (hunk 1 above)
Comment (line47) [reply docs/src/components/components/action/button/README.md]: Awesome, thanks!

Comment (line47) [reply docs/src/components/components/action/button/README.md]: @thisisdano what do you think here?
The class `action-cautious` is being used for the disabled state of the buttons and links. I think this class needs to be updated to `action-disabled`.

Comment (thisisdano) [reply docs/src/components/components/action/button/README.md]: agree that `action-cautious` is confusing. `action-disabled` is far clearer.

Comment (thisisdano) [reply docs/src/components/components/action/button/README.md]: it looks like the `disabled` prop renders `usa-button-disabled` in the templates in addition to any `style` props passed

Comment (line47) [reply docs/src/components/components/action/button/README.md]: I'll address that in a different PR. @thisisdano can you review this PR again?

**Comment by line47** [comment]
Adding [content guidelines](https://github.com/18F/cg-product/blob/master/ContentGuide.md) to the pattern library @brittag

**Review by thisisdano — APPROVED** [review]
lgtm

**Pull request merged**